src/openrct2/rct12/RCT12.cpp

```diff
@@ -511,8 +511,8 @@ bool RCT12ResearchItem::IsRandomEndMarker() const
 
 ObjectEntryIndex RCTEntryIndexToOpenRCT2EntryIndex(const RCT12ObjectEntryIndex index)
 {
-    if (index == RCT12_OBJECT_ENTRY_INDEX_NULL)
-        return OBJECT_ENTRY_INDEX_NULL;
+    if (index == RCT12_kObjectEntryIndexNull)
```

Comment:
This looks like it might have been an unintentional change. Perhaps we can put the alternate constant in the RCT12 namespace?